src/gui/StelGuiItems.cpp

```diff
@@ -774,15 +774,18 @@ void BottomStelBar::updateText(bool updatePos, bool updateTopocentric)
 	}
 
 	QString planetName = core->getCurrentLocation().planetName;
-	QString planetNameI18n;
+	if (planetName!=planetNameEnglish)
+	{
```

Comment:
This change leaves indentation broken.

Reply:
Ah yes. I was just so stunned by the effect of this single change that I just committed. Will fix that and rebase -i .